QueryManager/src/main/java/eu/neclab/ngsildbroker/queryhandler/repository/QueryDAO.java

```diff
@@ -800,7 +800,22 @@ public Uni<Tuple2<Map<String, Map<String, Object>>, List<String>>> queryForEntit
 			if (attrsQuery != null) {
 				dollar = attrsQuery.toSqlConstructEntity(query, tuple, dollar);
 			} else {
-				query.append("ENTITY.ENTITY");
+				if(qQuery!=null && qQuery.getAttribute().contains(NGSIConstants.QUERY_PARAMETER_DATA_SET_ID)){
+					query.append("""
+                            (jsonb_strip_nulls(jsonb_build_object(
+                            '@id', entity -> '@id',
+                            '@type', entity ->'@type',
+                            '@context', entity ->'@context',
+                            'https://uri.etsi.org/ngsi-ld/scope', entity ->'https://uri.etsi.org/ngsi-ld/scope',
+                            'https://uri.etsi.org/ngsi-ld/createdAt', entity ->'https://uri.etsi.org/ngsi-ld/createdAt',
```

Comment:
"https://uri.etsi.org/ngsi-ld/createdAt" so please take it from NGSIContants class.